x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1141,6 +1141,25 @@ describe('when on the endpoint list page', () => {
       );
     });
 
+    it('hides isolate host option if canIsolateHost is false', () => {
+      (useUserPrivileges as jest.Mock).mockReturnValue({
```

Comment:
Can you check if you need to do this?

If you look at:

https://github.com/elastic/kibana/blob/6efef0496077f4e61d49e4e43f75941ff3d98d9e/x-pack/plugins/security_solution/public/common/components/user_privileges/__mocks__/index.ts

All of the Endpoint authz Properties should already be set to `true` because we have a mock to set here: 

https://github.com/elastic/kibana/blob/6efef0496077f4e61d49e4e43f75941ff3d98d9e/x-pack/plugins/security_solution/public/common/components/user_privileges/endpoint/mocks.ts#L11-L21